src/analyzer/protocol/irc/IRC.cc

```diff
@@ -436,7 +436,7 @@ void IRC_Analyzer::DeliverStream(int length, const u_char* line, bool orig)
 				// Remove nick name.
 				parts.erase(parts.begin());
 
-				if ( parts.size() > 0 && parts[0][0] == ':' )
+				if ( parts[0][0] == ':' )
```

Comment:
I'm not immediately following why this is safe to do — perhaps context makes it a given.

Reply:
We already checked to see if `parts.size()` is < 2, and bailed out. We then removed an element which guarantees that the size is at least `1` here. I removed the check because it doesn't actually do anything,